src/lambda/lambdaTreeDataProvider.ts

```diff
@@ -11,20 +11,21 @@ import { RegionProvider } from '../shared/regions/regionProvider'
 import { ResourceFetcher } from '../shared/resourceFetcher'
 import { TelemetryNamespace } from '../shared/telemetry/telemetryTypes'
 import { registerCommand } from '../shared/telemetry/telemetryUtils'
-import { AWSCommandTreeNode } from '../shared/treeview/awsCommandTreeNode'
-import { AWSTreeNodeBase } from '../shared/treeview/awsTreeNodeBase'
 import { RefreshableAwsTreeProvider } from '../shared/treeview/awsTreeProvider'
+import { AWSCommandTreeNode } from '../shared/treeview/nodes/awsCommandTreeNode'
+import { AWSTreeNodeBase } from '../shared/treeview/nodes/awsTreeNodeBase'
+import { ErrorNode } from '../shared/treeview/nodes/errorNode'
+import { RegionNode } from '../shared/treeview/nodes/regionNode'
+import { showErrorDetails } from '../shared/treeview/webviews/showErrorDetails'
 import { intersection, toMap, updateInPlace } from '../shared/utilities/collectionUtils'
 import { localize } from '../shared/utilities/vsCodeUtils'
 import { deleteCloudFormation } from './commands/deleteCloudFormation'
 import { deleteLambda } from './commands/deleteLambda'
 import { invokeLambda } from './commands/invokeLambda'
-import { showErrorDetails } from './commands/showErrorDetails'
 import { CloudFormationStackNode } from './explorer/cloudFormationNodes'
+// tslint:disable-next-line:ordered-imports
```

Comment:
Is this disable lint statement necessary?

Reply:
Hmm, I think that was added by accident. Will fix.